aio/aio-proxy/aio_proxy/search_functions.py

```diff
@@ -0,0 +1,155 @@
+from elasticsearch_dsl import query
+
+
+def search_by_name(index, query_terms: str, offset: int, page_size: int):
+    s = index.search()
+    s = s.query(
```

Comment:
It would be interesting to comment it with different steps that we are implementing in it. It would be easier for someone new to understand what this query is doing.

Reply:
Yes, I agree! I have a ticket in my backlog for documenting it all because I think it's a pretty serious task and could take a while.
Is it okay with you if I do that separately on another PR, later on ?

Reply:
yes no pb